plugin.js

```diff
@@ -49,7 +56,7 @@ function Plugin (parent, func, optsOrFunc, isAfter, timeout) {
   this.timeout = timeout === undefined ? parent._timeout : timeout
   this.name = getName(func, optsOrFunc)
   this.isAfter = isAfter
-  this.q = fastq(parent, loadPlugin, 1)
+  this.q = fastq(parent, loadPluginNextTick, 1)
```

Comment:
I realize that delaying the loading until the next tick somewhat defeats the purpose of using a "fast" queue here, but this was the least invasive way I could find to keep things loading in the right order while also fixing the bug described in [fastify/3726](https://github.com/fastify/fastify/issues/3726).

Reply:
Also, I assumed that incurring a few extra ticks in the boot sequence wouldn't be a huge deal since it's a "one and done" kind of deal, i.e., not on the critical path for request execution.  (but please correct me if that's a bad assumption)

Reply:
> Also, I assumed that incurring a few extra ticks in the boot sequence wouldn't be a huge deal since it's a "one and done" kind of deal, i.e., not on the critical path for request execution. (but please correct me if that's a bad assumption)

I agree, but others are very concerned about boot speed. Are we able to quantify the difference? Something along the lines of https://github.com/jsumners/fjs-ajv-experiment should suffice.

Reply:
> Are we able to quantify the difference? Something along the lines of https://github.com/jsumners/fjs-ajv-experiment should suffice.

sure - I'll take a look at adapting your repo to quantify the perf difference

Reply:
I still need to convince myself that [my benchmark's plugin registrations](https://github.com/chrskrchr/fastify-avvio-benchmark/blob/main/index.js#L9-L18) are representative of what happens in the real world (so please take these with a grain of salt for now), but my [benchmark script](https://github.com/chrskrchr/fastify-avvio-benchmark) suggests that the overhead added with this change is minimal - less than `0.5%`.

#### Run #1 (+0.47%)
```
┌──────────┬─────────┬──────────┬──────────┬─────────────┬────────────┐
│ (index)  │ samples │ min (ms) │ max (ms) │ median (ms) │ total (ms) │
├──────────┼─────────┼──────────┼──────────┼─────────────┼────────────┤
│ current  │   500   │    33    │    82    │     40      │   20124    │
│ proposed │   500   │    35    │    70    │     40      │   20218    │
└──────────┴─────────┴──────────┴──────────┴─────────────┴────────────┘
```

#### Run #2 (+0.42%)
```
┌──────────┬─────────┬──────────┬──────────┬─────────────┬────────────┐
│ (index)  │ samples │ min (ms) │ max (ms) │ median (ms) │ total (ms) │
├──────────┼─────────┼──────────┼──────────┼─────────────┼────────────┤
│ current  │   500   │    32    │    77    │     37      │   18612    │
│ proposed │   500   │    31    │    56    │     37      │   18691    │
└──────────┴─────────┴──────────┴──────────┴─────────────┴────────────┘
```

#### Run #3 (+0.27%)
```
┌──────────┬─────────┬──────────┬──────────┬─────────────┬────────────┐
│ (index)  │ samples │ min (ms) │ max (ms) │ median (ms) │ total (ms) │
├──────────┼─────────┼──────────┼──────────┼─────────────┼────────────┤
│ current  │   500   │    33    │    83    │     39      │   19473    │
│ proposed │   500   │    33    │    62    │     39      │   19525    │
└──────────┴─────────┴──────────┴──────────┴─────────────┴────────────┘
```

Reply:
Looks good to me.